apollo-federation/tests/query_plan/build_query_plan_tests/handles_operations_with_directives.rs

```diff
@@ -244,7 +244,7 @@ fn test_if_directives_with_arguments_applied_on_queries_are_ok() {
 }
 
 #[test]
-#[should_panic(expected = r#"unused variable: `$some_var`"#)]
+#[should_panic(expected = r#"snapshot assertion"#)]
```

Comment:
The expected error changed because apollo-compiler fixed validation. This test still fails for other reasons, which will be addressed in https://github.com/apollographql/router/pull/5743.